pyproject.toml

```diff
@@ -94,5 +86,44 @@ exclude_also = [
     "@(abc\\.)?abstractmethod",
     "raise NotImplementedError",
     "\\.\\.\\.",
-    "pass",
+    "\\bpass$",
+]
+
+[tool.pyright]
+include = [
+    "upgrade_check",
+    "testapp",
+    "tests",
+]
+ignore = []
+pythonVersion = "3.12"
+pythonPlatform = "Linux"
+
+[tool.ruff.lint]
+extend-select = [
+    "UP",  # pyupgrade
+    "DJ",  # django
+    "LOG", # logging
+    "G",
+    "I",   # isort
+    "E",   # pycodestyle
+    "F",   # pyflakes
+    "PERF",# perflint
+    "B006",# flake8-bugbear
+    "B010",
+    "B904",
+]
```

Comment:
picked the config from OIP/django-upgrade-check, @swrichards is working on a nitpick solution to keep those in sync/centralized